index.html

```diff
@@ -2,9 +2,37 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <title>02-Box-Model</title>
+    <link rel="stylesheet" href="./app.css">
   </head>
   <body>
+    <div class="navbar">
+      <h1 class="nav_bar-left">Popular Github</h1>
+
+      <h2 class="nav_bar-right">People</h2>
```

Comment:
The right side of the nav will be easier to work with if you group them together and then the group can be floated right. This also helps keep the items in order.